moped-database/views/project_list_view.sql

```diff
@@ -89,7 +87,7 @@ AS WITH project_person_list_lookup AS (
     work_activities.task_order_names_short,
     work_activities.task_orders,
     'placeholder text' as project_development_status,
-    '2024-01-01T00:00:00−06:00' as project_development_status_date,
+    '2024-01-01T00:00:00-06:00' as project_development_status_date,
```

Comment:
Woa, that's subtle! 😵‍💫

Reply:
subtle for sure!!

Reply:
my IDE tried to warn me!

<img width="975" alt="thing" src="https://github.com/cityofaustin/atd-moped/assets/14793120/37cb2389-1e27-4d26-9150-a1deafe2dcd4">